testnets/mantrachaintestnet2/chain.json

```diff
@@ -0,0 +1,146 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "mantradukongtestnet",
```

Comment:
this value has to also be updated to mantrachaintestnet2

Reply:
My mistake, I just updated it, thanks for catching 🙏